### PR TITLE
Experiment 3 Implementation

### PR DESCRIPTION
**Pull request by rockretep**
Start of work on an implementation for experiment 3. Needs iteration to get intended behavior.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ For the contest the focus was on 4 substrates and their corresponding validation
 | | commons_harvest__partnership_5 |
 | commons_harvest__open | commons_harvest__open_0 |
 | | commons_harvest__open_1 |
+| | commons_harvest__farmer |
```
Comment (SoyGema) [inline README.md]: SUGGESTION : definition and 2 evaluation scenarios
```suggestion
| | commons_harvest__farmer | commons_harvest__open_0 |
| | commons_harvest__farmer_1 |
```

File: baselines/train/configs.py [hunk 2]
```diff
@@ -42,6 +42,7 @@
     'commons_harvest__private_property_1',
     'commons_harvest__open_disable_zapping_0',
     'commons_harvest__open_disable_zapping_1',
+    'commons_harvest__farmer_0',
```
Comment (SoyGema) [inline baselines/train/configs.py:49]: We are putting at least two eval scenarios per substrate
```suggestion
    'commons_harvest__farmer_0',
     'commons_harvest__farmer_1',
```

File: baselines/train/run_ray_train.py
Comment (SoyGema) [inline baselines/train/run_ray_train.py:1]: Revised : looks good to me

**Review by SoyGema — COMMENTED** [review]
A small review on README.md and add 2 evaluation scenarios .
Keep it up!

**Comment by whymath** [comment]
@rockretep please pull the latest from the main branch to account for the changes from Chris' PR #4 and resolve the merge conflicts (along with any pending changes from Gema's review)

**Comment by whymath** [comment]
@rockretep heads up that all other open PRs have been merged, so you should be able to take the latest from the main branch now

**Comment by rockretep** [comment]
Everything should be ready to merge into main now.

**Comment by SoyGema** [comment]
Thanks for your efforts @rockretep !🙂

File: README.md [hunk 3]
```diff
@@ -49,6 +49,8 @@ For the contest the focus was on 4 substrates and their corresponding validation
 | | commons_harvest__partnership_5 |
 | commons_harvest__open | commons_harvest__open_0 |
 | | commons_harvest__open_1 |
+| | commons_harvest__farmer | commons_harvest__open_0 |
```
Comment (whymath) [inline README.md:52]: First scenario should be "__farmer_0"

**Review by whymath — APPROVED** [review]
Minor readme update pending, but that can be updated later, so good to merge

**Pull request merged**